mountpoint-s3/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 ## Unreleased
 
+## v1.6.0 (March 11, 2024)
+
 ### New features
 * Mountpoint for Amazon S3 now supports specifying an AWS Key Management Service (AWS KMS) key for server-side encryption with KMS (SSE-KMS) when mounting an S3 bucket or prefix. ([#839](https://github.com/awslabs/mountpoint-s3/pull/839))
 
+### Breaking changes
+* No breaking changes.
+
 ### Other changes
 
 * Mountpoint now retries S3 requests up to a total of 10 attempts (up from 4), which should make file operations more robust to transient failures or throttling. The maximum number of attempts can be overridden by setting the `AWS_MAX_ATTEMPTS` environment variable. ([#830](https://github.com/awslabs/mountpoint-s3/pull/830))
```

Comment:
- Adopt new async write API for PutObject requests: https://github.com/awslabs/mountpoint-s3/commit/2767c856d96984a523aad6a92f0f322e461ccdaf
- Adding support for rewinddir by restarting readdir if offset is zero: https://github.com/awslabs/mountpoint-s3/commit/127fb714e3b279291dacc6f59e9c6291fbf5a611